src/main/java/me/kavin/piped/utils/obj/CommentsPage.java

```diff
@@ -7,10 +7,13 @@ public class CommentsPage {
     public List<Comment> comments;
     public String nextpage;
     public boolean disabled;
+    // The comment count is not available for nextpage responses
+    public Integer commentCount;
 
-    public CommentsPage(List<Comment> comments, String nextpage, boolean disabled) {
+    public CommentsPage(List<Comment> comments, String nextpage, boolean disabled, Integer commentCount) {
         this.comments = comments;

```

Comment:
Please use the primitive `int` type instead, otherwise LGTM!

Reply:
That doesn't work unfortunately because int is not nullable :(

Reply:
Yes, but `-1` could be provided instead of null.